mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -141,37 +149,37 @@ def preprocess_mpas(ds, onlyvars=None, selvals=None, iselvals=None,
     timestr='time_avg_daysSinceStartOfSim' and for MPAS-Seaice
     timestr='timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'.
 
-    The onlyvars option reduces the dataset to only include variables in the onlyvars list.
-    If onlyvars=None, include all dataset variables.
+    The onlyvars option reduces the dataset to only include variables in the
+    onlyvars list. If onlyvars=None, include all dataset variables.
 
-    iselvals and selvals provide index and value-based slicing operations for individual datasets
-    prior to their merge via xarray.
-    iselvals is a dictionary, e.g., iselvals = {'nVertLevels': slice(0,3), 'nCells': cellIDs}
+    iselvals and selvals provide index and value-based slicing operations for
+    individual datasets prior to their merge via xarray.
+    iselvals is a dictionary, e.g., iselvals = {'nVertLevels': slice(0,3),
+                                                'nCells': cellIDs}
     selvals is a dictionary, e.g., selvals = {'cellLon': 180.0}
 
-    Phillip J. Wolfram, Milena Veneziani, and Luke van Roekel
-    09/13/2016
+    Phillip J. Wolfram, Milena Veneziani, Luke van Roekel and Xylar Asay-Davis
+    11/25/2016
     """
 
     # ensure timestr is specified used when timeSeriesStats=True
     if timeSeriesStats:
         if timestr is None:
             assert False, 'A value for timestr is required, e.g., ' + \
                     'for MPAS-O: time_avg_daysSinceStartOfSim, and ' + \
-                    'for MPAS-Seaice: timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'
+                    'for MPAS-Seaice: ' + \
+                    'timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'
 
         # compute shifted datetimes
         daysSinceStart = ds[timestr]
-        datetimes = [datetime.datetime(yearoffset, monthoffset, dayoffset) + x
+        datetimes = [datetime.datetime(yearoffset+1, 1, 1) + x
```

Comment:
Please see comment above about hard coding values.  I think we only want the default to be changed to avoid hard-coded "magic" numbers.